src/mixins/common-mixin.ts

```diff
@@ -9,9 +9,9 @@ import { getName } from 'country-list'
  */
 @Component({})
 export default class CommonMixin extends Vue {
-  /** Is True if Jest is running the code. */
-  get isJestRunning (): boolean {
-    return (process.env.JEST_WORKER_ID !== undefined)
+  /** Is True if Vitest is running the code. */
+  get isVitestRunning (): boolean {
+    return (import.meta.env.VITEST_WORKER_ID !== undefined)
```

Comment:
Check Vitest Worker Id is probably fine, but I found an env key simply called VITEST which is either `'true'` or absent (undefined). Did you see somewhere where it was recommended to use the worked ID?

Reply:
Please check my comment above Sev.